### PR TITLE
Router CI Jenkinsfile

### PR DESCRIPTION
**Pull request by djsd123**
Replaced router bash scripts with groovy script for Jenkins

The Jenkinsfile conflates building and updating the GitHub status of the application

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            }
+        }
+
+        // Start Build
+        stage("Build") {
+            dir(env.SRC_PATH) {
+                sh 'BINARY=$WORKSPACE/router make clean build test'
+            }
+
+            dir(env.GOPATH)
```
Comment (djsd123) [inline Jenkinsfile]: Add `{}` to this to form a block.  Change to `SRC_PATH`.

**Comment by djsd123** [comment]
Review label definitions of the agent nodes.  i.e. mongodb-2.4 etc....

**Comment by surminus** [comment]
:+1: This all looks fine to me, feel free to merge

**Comment by djsd123** [comment]
@surminus Ta

**Pull request merged**